src/components/Quest.vue

```diff
@@ -163,12 +163,13 @@ export default {
 #quest {
   background: var(--color-blue);
   position: relative;
+  align-items: center;
```

Comment:
Nie do końca rozumiem, jaki związek ma ta zmiana w związku z przyciskiem, ale rozumiem że to jest powiązane z ustawianiem `align-items: left;` dla `#app > div`.

Reply:
tak jest